File: app/src/main/java/com/condorlabs/popularmovies/databinding/ImageBindingAdapter.java
package com.condorlabs.popularmovies.databinding;

import android.databinding.BindingAdapter;
import android.widget.ImageView;

import com.condorlabs.popularmovies.util.AppConstants;
import com.squareup.picasso.Picasso;

public final class ImageBindingAdapter {

    @BindingAdapter(value = "url")
    public static void loadPictureUrl(ImageView view, String url) {
        if (url != null && !url.equals(""))
            Picasso.get()
                    .load(AppConstants.PICTURES_URL + url)
                    .into(view);
    }

}

File: app/src/main/java/com/condorlabs/popularmovies/databinding/ImageBindingAdapter.kt
package com.condorlabs.popularmovies.databinding

import android.databinding.BindingAdapter
import android.widget.ImageView

import com.condorlabs.popularmovies.util.AppConstants
import com.squareup.picasso.Picasso

object ImageBindingAdapter {

    @BindingAdapter(value = ["url"])
    fun loadPictureUrl(view: ImageView, url: String?) {
        if (url != null && url != "")
            Picasso.get()
                    .load(AppConstants.PICTURES_URL + url)
                    .into(view)
    }

}
